src/components/inputs/InputText/index.tsx

```diff
@@ -61,7 +61,7 @@ function InputText(
       </InputContainer>
 
       {/* Show error Messages */}
-      {error && <ErrorMessage text={error} />}
+      {error && <ErrorMessage color="redGraphOrError" text={error} />}
```

Comment:
remove color, not used

Reply:
I think this story is a bit confusing. It's says error, but is really a notification text